src/lib/repositories_v2/order_repository.py

```diff
@@ -0,0 +1,22 @@
+# Interface for the repositories through Abstract method
+from abc import ABCMeta
+
+from src.lib.repositories.generic_repository import GenericRepository
+
+
+class OrderRepository(GenericRepository, metaclass=ABCMeta):
```

Comment:
You don't need to copy and paste this if it already exists, the same for the other existing repositories

Reply:
i create a new one in the cases where i dont have the same methods(functions) as version v1 of the project.

Reply:
This part will be fixed in a refactor PR.